src/consensus/consensus.h

```diff
@@ -8,6 +8,12 @@
 
 #include <stdint.h>
 
+/**
+ * Maximum amount of time that a block timestamp is allowed to exceed the
+ * current network-adjusted time before the block will be accepted.
+ */
+static const int64_t MAX_FUTURE_BLOCK_TIME = 2 * 60 * 60;
```

Comment:
In commit "MOVEONLY: MAX_FUTURE_BLOCK_TIME to consensus/consensus.h"

Not sure if if it's exactly right to make this a consensus constant, because even in the worst case if this is set inconsistently, consensus should eventually be reached when system clock times catch up. IIRC, this was the reasoning @morcos gave me for not defining this constant in consensus.h.